### ISSUE TITLE
Exo-Striker crashes randomly with  "Segmentation fault (core dumped)" (HELP NEEDED)

### ISSUE THREAD

**Issue by 3fon3fonov** [3fon3fonov/exostriker#128]
Unfortunately, since the fortran RV binaries were translated to .so libs with f2py, in some machines the Exo-Striker crashes randomly with:

`$ Segmentation fault (core dumped)
`
It has been observed on Python3.8, 3.9, 3.10, and 3.11 and many versions gfortran, PyQt5, and numpy. 

The best I could trace on Ubuntu 22.04 Intel-based configuration was:

```
$ trifonov@lnx-n-0265:~/exostriker-master$ gdb -ex r --args python3.10 -m exostriker
$ (gdb)
$ [Thread debugging using libthread_db enabled]
$ Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
$ [New Thread 0x7ffff2dff640 (LWP 839683)]
$ ...
$ ...
$ Thread 1 "python3.10" received signal SIGSEGV, Segmentation fault.
0x00007fff9f6bd91a in QFontEngineFT::loadGlyphSet(QTransform const&) () from /home/trifonov/.local/lib/python3.10/site-packages/PyQt5/Qt5/plugins/platforms/../../lib/libQt5XcbQpa.so.5
(gdb)

```

Which points to memory issue with one of the Qt5 libraries. 

However, I am not entirely sure the problem is in PyQt5/Qt5. It could be a memory issue with the [rvmod_for.f95](https://github.com/3fon3fonov/exostriker/blob/exostriker-master/exostriker/lib/RV_mod/rvmod/rvmod_for.f95) source file, although very extensive tests on the importable code (rvmod_*.so) could not reproduce the error.

For now, I will try to push an urgent PyQt6-upgrade and see if the problem is indeed in PyQt. Else, I am afraid more time must be spend on the fortran code to make sure it its variables are not causing memory issues. 

If you have similar instability problems, please report your system configurations, as explained here: https://github.com/3fon3fonov/exostriker/issues/60

**Comment by 3fon3fonov** [comment]
A new upgrade of the Exo-Striker with PyQt6 is on its way as of 12 Feb. 2024. It seems to be much more stable on my linux Machine with Ubuntu22.04. However, not without problems..... I tried to reproduce the crash above, by minimizing the GUI over few hundred times and everything worked fine until I moved slightly the window... Then I got another cryptic error in :

```

Thread 1 "python3.11" received signal SIGSEGV, Segmentation fault.
0x00007fffd3a1623c in QFontEngineFT::loadGlyphSet(QTransform const&) () from /home/trifonov/.local/lib/python3.11/site-packages/PyQt6/Qt6/lib/libQt6Gui.so.6
(gdb) 

```

I am more and more convinced that this PqQt problem, particularly with some of the QFonts, but it is likely that a bug in my code is invoking the crash. The 
 bug hunt continues....
